classes/local/store/s3/client.php

```diff
@@ -37,6 +37,14 @@
 define('AWS_CAN_DELETE_OBJECT', 2);
 
 class client extends object_client_base {
+
+    /**
+     * @var int A predefined limit of data stored.
+     * When hit, php://temp will use a temporary file.
+     * Reference: https://www.php.net/manual/en/wrappers.php.php
```

Comment:
Can you please link to this:

https://github.com/catalyst/moodle-local_aws/blob/master/sdk/Aws/S3/StreamWrapper.php#L19-L25